pkg/executor/importer/import.go

```diff
@@ -810,13 +811,11 @@ func (p *Plan) initParameters(plan *plannercore.ImportInto) error {
 		setClause = sb.String()
 	}
 	optionMap := make(map[string]any, len(plan.Options))
-	var evalCtx expression.EvalContext
-	if plan.SCtx() != nil {
-		evalCtx = plan.SCtx().GetExprCtx().GetEvalCtx()
-	}
 	for _, opt := range plan.Options {
 		if opt.Value != nil {
-			val := opt.Value.StringWithCtx(evalCtx, errors.RedactLogDisable)
+			var val string
+			cons := opt.Value.(*expression.Constant)
+			val = fmt.Sprintf("%v", cons.Value.GetValue())
```

Comment:
```suggestion
			cons := opt.Value.(*expression.Constant)
			val := fmt.Sprintf("%v", cons.Value.GetValue())
```

Also please add some comment about why it is safe to assert opt.Value is a constant.